src/components/Settings/AdvancedSettings.tsx

```diff
@@ -0,0 +1,49 @@
+import { TextInput, Group, Button } from '@mantine/core';
+import { useState } from 'react';
+import { useConfig } from '../../tools/state';
+
+export default function TitleChanger() {
+  const { config, loadConfig, setConfig, getConfigs } = useConfig();
+  const [customTitle, setCustomTitle] = useState(config.title);
```

Comment:
```suggestion
  const [customTitle, setCustomTitle] = useState(config.title ?? 'Homarr 🦞');
```
Or something like that would fix the default state for the custom title.